script.js

```diff
@@ -82,27 +82,18 @@ function valutaConverter(event) {
 // **********************************************
 // An array of the currency rate objects
 // **********************************************
-
-const brandObjectArray = [
-  {
-    //timestamp: Date.now(),
-    base: "EUR",
-    date: new Date().toLocaleDateString(),
-    rates: {},
-  },
-  {
-    //timestamp: Date.now(),
-    base: "USD",
-    date: new Date().toLocaleDateString(),
-    rates: {},
-  },
-  {
-    //timestamp: Date.now(),
-    base: "DKK",
-    date: new Date().toLocaleDateString(),
-    rates: {},
-  },
-];
+const brandObjectArray = [];
+fetch(
+  "https://raw.githubusercontent.com/katayouny/katayouny.github.io/main/data/currency-converter.json"
+)
+  .then((response) => response.json())
+  .then((json) => {
+    brandObjectArray.push(...json);
+    setEvents();
+  })
+  .catch((error) => {
+    setErrors(error);
+  });
```

Comment:
nice job @katayouny I like the error handling :)

Reply:
Thank you @BrantalikP :)